src/lib/draw-and-pick.js

```diff
@@ -196,6 +210,17 @@ export function pickLayers(gl, {
     if (info) {
       infos.set(info.layer.id, info);
     }
+
+    const pickingSelectedColor = pickedColor;
+    const pickingValid = Boolean(
```

Comment:
We have two types of picking now, custom and auto highlight, and when both set custom picking takes precedence as discussed in RFC. This logic needed to handle all possible corner cases.
And all this happens inside of deck.gl, core layers and applications using this feature only deal with setting the props, lets discuss if any additional changes needed in this regard.